From fd3d29171d783e4492f78387ecc4f3cb5908819c Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Fri, 15 Sep 2017 09:34:17 +0800
Subject: [PATCH] phy: fix Kconfig dependencies for PHY_ROCKCHIP_INNO_USB2

Rockchip Inno usb2-phy uses extcon_* methods and because of
that, it must select EXTCON as a dependency. This patch fixes it.

Change-Id: Ia2b712154e4fee567b843f38f50fbcc00c49bf1e
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 drivers/phy/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/phy/Kconfig b/drivers/phy/Kconfig
index 1f4c536293f3..3fab54e7920c 100644
--- a/drivers/phy/Kconfig
+++ b/drivers/phy/Kconfig
@@ -325,6 +325,7 @@ config PHY_ROCKCHIP_INNO_USB2
 	depends on ARCH_ROCKCHIP && OF
 	depends on COMMON_CLK
 	depends on USB_SUPPORT
+	select EXTCON
 	select GENERIC_PHY
 	select USB_COMMON
 	help
-- 
2.35.3

